packages/contracts/contracts/MinimumViableMultisig.sol

```diff
@@ -34,7 +34,8 @@ contract MinimumViableMultisig {
 
   /// @notice Contract constructor
   /// @param owners An array of unique addresses representing the multisig owners
-  constructor(address[] owners) public {
+  function setup(address[] owners) public {
```

Comment:
If we go with this change, we should add a comment explaining how it's meant to be deployed through `ProxyFactory` . Otherwise, we have to incur two transactions to setup the multisig instead of one.

Reply:
👍